lib/internal/test_runner/utils.js

```diff
@@ -128,10 +129,17 @@ function tryBuiltinReporter(name) {
   return require(builtinPath);
 }
 
-async function getReportersMap(reporters, destinations, rootTest) {
+function colorizeTestFiles(rootTest) {
+  const { reporters, destinations } = parseCommandLine();
+  ArrayPrototypeForEach(reporters, (_, index) => {
+    const destination = kBuiltinDestinations.get(destinations[index]);
+    rootTest.harness.shouldColorizeTestFiles ||= shouldColorize(destination);
+  });
+}
```

Comment:
```suggestion
function shouldColorizeTestFiles(rootTest) {
  const { reporters, destinations } = parseCommandLine();
  return ArrayPrototypeSome(reporters, (_, index) => {
    const destination = kBuiltinDestinations.get(destinations[index]);
    return shouldColorize(destination);
  });
}
```

Reply:
we should also add a test and make sure the beavior is preserved for 
`FORCE_COLOR=1 node --test --test-reporter spec --test-reporter-destination spec.txt test.js`

Reply:
like this https://github.com/nodejs/node/pull/50982/files#diff-af9e4636217bd4613d5bea8d7080880d0a291af0e8de848dfa401caf5e76a6bfR158-R176 ???